ReleaseChecklist.md

```diff
@@ -2,28 +2,32 @@
 
 ### Requirements
  - [ ] Lauchpad (Ubuntu One) account
- - [ ] gnupg key (has to be version 1, gpg2 won't work) for `your-name@ethereum.org` created and uploaded
- - [ ] Readthedocs account, access to the Solidity project
+ - [ ] gnupg key (has to be version 1, gpg2 won't work) for ``your-name@ethereum.org`` created and uploaded
  - [ ] Write access to https://github.com/ethereum/homebrew-ethereum
 
-### Documentation check
- - [ ] Run `make linkcheck` from within `docs/` and fix any broken links it finds. Ignore false positives caused by `href` anchors and dummy links not meant to work.
-
 ### Blog Post
- - [ ] Create a post on https://github.com/ethereum/solidity-blog and explain some of the new features or concepts.
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) in the ``Releases`` category and explain some of the new features or concepts.
+ - [ ] Create a post on [solidity-blog](https://github.com/ethereum/solidity-blog) in the ``Security Alerts`` category for each important bug.
```

Comment:
A blog post PR actually... and "for each important bug" is rather pessimistic :-D... ``in case of important bug(s)`` rather?

Reply:
Haha, it's technically correct but I guess we can make it less scary :) Changed.